### PR TITLE
Fix missing "write-plugin" HOCON settings

### PR DESCRIPTION
**Pull request by Arkatufus**
Fix missing "write-plugin" HOCON settings

## Changes

* Generate "akka.persistence.query.journal.sql.write-plugin" setting
* Port `Bugfix344Spec` to Hosting.Tests as regression test

## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] I have [reviewed my own pull request](https://getakka.net/community/contributing/index.html#review-your-own-pull-requests).

### CODE REVIEW

File: src/Akka.Persistence.Sql.Hosting/SqlJournalOptions.cs [hunk 1]
```diff
@@ -154,19 +154,20 @@ protected override StringBuilder Build(StringBuilder sb)
...
             return sb;
         }
 
-        private StringBuilder BuildQueryConfig(StringBuilder sb, string queryPluginId)
+        private StringBuilder BuildQueryConfig(StringBuilder sb, string queryPluginId, string pluginId)
         {
             sb.Append(queryPluginId).AppendLine("{");
             sb.AppendLine($"connection-string = {ConnectionString.ToHocon()}");
             sb.AppendLine($"provider-name = {ProviderName.ToHocon()}");
+            sb.AppendLine($"write-plugin = {pluginId}");
```
Comment (Arkatufus) [inline src/Akka.Persistence.Sql.Hosting/SqlJournalOptions.cs:170]: This is the missing setting.

File: src/Akka.Persistence.Sql/Config/ReadJournalConfig.cs [hunk 2]
```diff
@@ -31,6 +32,8 @@ public ReadJournalConfig(Configuration.Config config)
             WriteIsolationLevel = IsolationLevel.Unspecified;
         }
 
+        public string WritePluginId { get; }
```
Comment (Arkatufus) [inline src/Akka.Persistence.Sql/Config/ReadJournalConfig.cs:35]: Adding the missing setting as a property in `ReadJournalConfig` as its supposed to

File: src/Akka.Persistence.Sql/Query/SqlReadJournal.cs [hunk 3]
```diff
@@ -52,15 +52,14 @@ public class SqlReadJournal :
             ExtendedActorSystem system,
             Configuration.Config config)
         {
-            var writePluginId = config.GetString("write-plugin");
```
Comment (Arkatufus) [inline src/Akka.Persistence.Sql/Query/SqlReadJournal.cs:55]: This is not deleted, we're using the new `WritePluginId` property in `ReadJournalConfig` instead.

**Review by Arkatufus — COMMENTED** [review]
Self-review

File: src/Akka.Persistence.Sql.Hosting/SqlJournalOptions.cs (hunk 1 above)
Comment (Aaronontheweb) [reply src/Akka.Persistence.Sql.Hosting/SqlJournalOptions.cs:170]: Nice catch

**Review by Aaronontheweb — APPROVED** [review]
LGTM - guess we'll need to do a v1.5.27.1 release

**Pull request merged**